### ISSUE TITLE
Upload file conflict persist independently on user choice

### ISSUE THREAD

**Issue by kofaysi** [nextcloud/android#6510]
### Steps to reproduce
1. Saved a Debug info into a folder
2. NC app tries to upload
3. Sync conflict dialogue comes up
4. user choose all option: keep local, keep server, keep both and rename

### Expected behaviour
- files is synced

### Actual behaviour
- file is uploaded with numerical increment (2), (3),... but sync conflict dialogue comes up again within an hour time

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version: 7.1.3

Device model: LG-H850

Stock or customized system: OSe

Nextcloud app version: Dev channel 20200721

Nextcloud server version: 18 ecloud.global

Reverse proxy:

### Logs
![Screenshot_20200722-002437](https://user-images.githubusercontent.com/17022173/88113886-fef04d80-cbb2-11ea-8cbc-ab40e18e3ba5.png)
The file creating conflict is to be supplied privately. It is a Debug info.
#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by kofaysi** [comment]
Yesterday, the time to the left file said 4 or "6 hour ago". The screenshot says year 1970 to my surprise.

**Comment by tobiasKaminsky** [comment]
This is indeed strange.
Do you see this file in upload queue?
Is this really on server?

**Comment by kofaysi** [comment]
It would be also nice to be able to see the filename, or/and open file preview by clicking on the thumbnail. It would help to check on details and decide on the sync action.

**Comment by tobiasKaminsky** [comment]
Filename is the same, as otherwise there would be no conflict ;-)
File preview is technically a bit challenging as you already have on dialog…
In such uncertain situations I would just save both and then review them in normal mode.

**Comment by kofaysi** [comment]
OK, thank for the note on the preview, dear Tobias. That is what I did: checked and saved both and reviewed and compared by download on the PC. In some time (1 hour?) the conflict appeared again. (And that is what this issue is about.)

**Comment by kofaysi** [comment]
The files indeed exist on both sides. I'm attaching screenshots about their existence on the mobile and at the cloud as well as some screenshots about their properties.

![Screenshot_20200722-105937_edit_edit](https://user-images.githubusercontent.com/17022173/88157931-aef1a500-cc0b-11ea-8900-f382d0237123.png)
![Screenshot_20200722-105611_edit_edit](https://user-images.githubusercontent.com/17022173/88157937-b022d200-cc0b-11ea-9927-151f91ce38fb.png)
![Screenshot_20200722-105541_edit](https://user-images.githubusercontent.com/17022173/88157938-b022d200-cc0b-11ea-865d-12216a4189c8.png)
![Screenshot_20200722-105453_edit](https://user-images.githubusercontent.com/17022173/88157940-b0bb6880-cc0b-11ea-964d-8f3c7b42d26d.png)
(Some identification names were disguised.)

The screenshot here states that all of the files with (#) extension were created 21 hours ago. Than should not be true. each (#) was created with at least 1 hour time difference.
![Screenshot_2020-07-22 Downloads - Files - e ](https://user-images.githubusercontent.com/17022173/88157695-63d79200-cc0b-11ea-96ba-1a1cfa2c7289.png)

In the morning the NC Dev app was updated to v20200722 and the conflict appeared right now, too.

I still keep the two files on the two places. I know that I can get rid off all files and start over, but I hope to help and shed light on the issue.

**Comment by kofaysi** [comment]
(It is indeed strange that I use NectCloud webinterface in English and Firefox Quantum in English locale, but the times on the list are named in Czech. This Czech settings may come from the Computer locate Date and Currency settings...?!)

**Comment by kofaysi** [comment]
> Filename is the same, as otherwise there would be no conflict ;-)

That is correct. But when analyzing any glitch or issue, one cannot see the filename on the conflict preview. The way to look the filename and files up is to cancel the conflict dialogue and find the most recent failed upload in the upload queue -- which at the time maybe is not the most recent, because other uploads could fails as well for other reasons more recently.

**Comment by tobiasKaminsky** [comment]
The (2) (3) only appear when you select "keep both" or?

**Comment by Queuecumber** [comment]
I think I'm also having this issue, it's extremely annoying.

I have the app auto uploading pictures, if I edit one of them then I get the conflict dialog, no matter what option I pick it bugs me again in a few minutes about a conflict. It's a recent breakage it was working fine for a long time. I have that folder set to "overwrite the remote version" in case of a conflict too so in theory I should never see that notification.

**Comment by Queuecumber** [comment]
Even if I turn auto uploading off for that folder I keep getting the notification

**Comment by tobiasKaminsky** [comment]
Fixed via #6771
